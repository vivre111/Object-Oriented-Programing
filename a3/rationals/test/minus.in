create a -1/5 create b 2/6 create c -0/2 create d 1/5
- a b - a c - a d - b d - d b print b print a - c c create a 9/3 print a
