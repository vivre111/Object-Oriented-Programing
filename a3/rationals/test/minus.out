-8/15
-1/5
-2/5
2/15
-2/15
1/3
-1/5
0
3
